Resources/Prototypes/Entities/Objects/Devices/EngBag.yml

```
@@ -0,0 +1,30 @@
+- type: entity
+  name: engineer bag
+  id: EngBag
```

better to keep a convenient name and id
```suggestion
  name: engineer bag
  id: EngBag
```